controllers/homeController.js

```diff
@@ -1,81 +1,80 @@
-function getHomeFeaturedScenes() {
-	var scenes = {}
+const when = require('when')
+const Graph = require('../models/graph')
+const fsPath = require('path')
 
-	function scene(username, path, ownerName, sceneName) {
-		return {
-			graphName: sceneName,
-			graphOwner: ownerName,
-			graphOwnerLink: "//vizor.io/" + username,
-			previewUrlSmall: "//vizor.io/data/previews/"+username+"/"+path+"-preview-440x330.png",
-			path: "//vizor.io/"+username+"/"+path
-		}
+function renderFeatured(graph) {
```

Comment:
p.s. since graphSmall.handlebars is only used there, it's safe to change the fields inside (e.g. prettyName, prettyOwner, etc, as required